pennylane/operation.py

```diff
@@ -559,28 +558,67 @@ def hyperparameters(self):
         self._hyperparameters = {}
         return self._hyperparameters
 
-    @staticmethod
-    def decomposition(*params, wires):
-        """Defines a decomposition of this operator into products of other operators.
+    def decomposition(self):
+        """The decomposition of the Operator into a product of more fundamental gates.
 
-        Args:
-            params (tuple[float, int, array]): operator parameters
-            wires (Union(Sequence[int], Wires)): wires the operator acts on
+        .. math:: O = O_1 O_2 \dots O_n
+
+        Returns ``None`` is this Operator does not define its decomposition.
```

Comment:
Just leaving comments where docstrings would need to change if the error is used instead of `None`